libs/feature/record/src/lib/record-metadata/record-metadata.component.ts

```diff
@@ -77,4 +83,9 @@ export class RecordMetadataComponent {
       .getFiltersForOrgs([{ name: contact.organisation }])
       .subscribe((filters) => this.searchService.updateFilters(filters))
   }
+
+  openLightbox(src: string) {
+    this.lightboxInstance = basicLightbox.create(`<img src="${src}"/>`)
```

Comment:
Is it cleaned somehow if you click several times ?

Reply:
Yes, the dom element is removed each time a click on background overlay is done.